content/material_and_questions.md

```diff
@@ -13,23 +13,31 @@
 | 2026-2027 | John                                          | Narrative |
 | 2027-2028 | Hebrews, 1 Peter, 2 Peter                     | Epistle   |
 
-## Question Writing
+## Material
+: The biblical content that is the basis for competition for the quiz year. This includes Book Name, Chapter Number, and Verse Number. This does not include any specifically excluded material.
 
-### Preparation of Questions
+Of the eight years of material, 4 are defined as Narrative and 4 are defined as Epistle.
 
-### Question Types
+## Interrogative Words
+: The single, interrogative words that must be inserted at either the beginning or the end of every [ INT, MA, CR, CVR ] question. The exhaustive list of these words is: who, whom, whose, what, why, where, when, which, how.
 
-#### Interrogative
+## Unique Word
+: A word appearing only once in the material or a word that occurs multiple times, but only in 1 verse in the material. Also known as "global unique words."  
 
-#### Multiple Answer
+## Unique Phrases
+: 2 or 3 consecutive words, the sequence of which is only found once in the material. While unique words have the "occurring more than once but in the same verse" corollary, this concept does not exist for 2-3 word phrases. 
 
-#### Reference
+## First 5 Words of the Question
+: Defined as "the first 5 spoken words after Question, whether the words are from the material or not." This concept applies to the beginning of INT, MA, F, SIT questions. 
```

Comment:
Should avoid saying "INT, MA, F, SIT" here since those terms haven't yet be defined.

Reply:
Agreed. It's just so much shorter.

Reply:
No, sorry. I mean that a "Situation" question hasn't been defined before here. So we have a choice to make: We can avoid saying "Situation" or "SIT" here; instead, maybe say "certain question types" and then below when we define each, reference back that the 5-words rule applies. Or we accept that sometimes there will be concepts first defined after their usage. Ideally, I'd like to go with the former rather than the latter, but it may be practically impossible without a lot of contortions, so maybe ROI-negative.

Reply:
hmmm, I see your conundrum. But I like the latter better. I don't want these "90% overlapping phrases" strewn/repeated throughout each Question Type definition.